playground/house.py

```diff
@@ -45,9 +45,31 @@ def play(self):
 
         self.board.render_figure()
 
+    def simulation(self):
```

Comment:
generally, it is recommended to use function names in verb form: https://stackoverflow.com/questions/1991324/function-naming-conventions